docs/development/tests.rst

```diff
@@ -7,26 +7,34 @@ and your code style passes our code linting suite.
 Read the Docs uses `Tox`_ to execute testing and linting procedures. Tox is the
 only dependency you need to run linting or our test suite, the remainder of our
 requirements will be installed by Tox into environment specific virtualenv
-paths. Before testing, make sure you have Tox installed::
+paths. Before testing, make sure you have Tox installed:
+
+.. prompt:: bash
 
     pip install tox
```

Comment:
We are running tests from inside docker compose setup now. It may be better to mention that here instead.

```
inv docker.test
inv docker.test -a '-e py36'
```

Reply:
I can't recommend that yet, I think. On my setup this takes a really long time to run.